src/nodeodm_client/nodeodm_client.py

```diff
@@ -0,0 +1,237 @@
+import asyncio
+import json
+import logging
+import logging.config
+import os
+
+import requests
```

Comment:
Ist das nicht eine externe Dependency? D.h. normalerweise sollte die in einer requirements.txt für PIP definiert sein:

https://github.com/vitrum-connect/5gla-sensor-integration-script/blob/main/requirements.txt